website/pages/docs/job-specification/hcl2/functions/index.mdx

```diff
@@ -0,0 +1,29 @@
+---
+layout: docs
+page_title: Functions - Configuration Language
+sidebar_title: Functions
+description: |-
+  The HCL language has a number of built-in functions that can be called
+  from within expressions to transform and combine values.
+---
+
+# Built-in Functions
+
+`@include 'beta-nomad1.0-note.mdx'`
+
+The HCL language includes a number of built-in functions that you can
+call from within expressions to transform and combine values. The general
+syntax for function calls is a function name followed by comma-separated
+arguments in parentheses:
+
+```hcl
+max(5, 12, 9)
+```
+
+For more details on syntax, see
+[_Function Calls_](/docs/job-specification/hcl2/expressions#function-calls)
```

Comment:
There's no such anchor on this page. https://nomad-7s3t20jvx.vercel.app/docs/job-specification/hcl2/expressions#available-functions just points back here.

Reply:
Removed the anchor - thanks